doc/src/build/snapshot.md

```diff
@@ -13,22 +13,22 @@ Full nodes do not take snapshots by default. To enable this feature:
 1. Stop your node, if it's running.
 2. Open your fullnode.yaml config file.
 3. Add an entry to the config file for `db-checkpoint-config`. Using Amazon's S3 service as an example:
-    ```yaml
-    db-checkpoint-config:
-      perform-db-checkpoints-at-epoch-end: true
-      object-store-config:
-        object-store: "S3"
-        bucket: "<BUCKET-NAME>"
-        aws-access-key-id: “<ACCESS-KEY>”
-        aws-secret-access-key: “<SHARED-KEY>”
-        aws-region: "<BUCKET-REGION>"
-        object-store-connection-limit: 20
-    ```
-    - `object-store`: The remote object store to upload snapshots. Set as Amazon's `S3` service in the example.
-    - `bucket`: The S3 bucket name to store the snapshots.
-    - `aws-access-key-id` and `aws-secret-access-key`: AWS authentication information with write access to the bucket.
-    - `aws-region`: Region where buck exists.
-    - `object-store-connection-limit`: Number of simultaneous connections to the object store.
+   ```yaml
+   db-checkpoint-config:
+     perform-db-checkpoints-at-epoch-end: true
```

Comment:
Should we also add the `perform index db snapshot` flag? I think it is opt-in and most FNs are likely going to want it.